File: autofin/src/main/java/model/add_lead_details/AddLeadResponse.java
package model.add_lead_details;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class AddLeadResponse {
    @SerializedName("status")
    @Expose
    private Boolean status;
    @SerializedName("message")
    @Expose
    private String message;
    @SerializedName("statusCode")
    @Expose
    private String statusCode;
    @SerializedName("data")
    @Expose
    private Integer data;

    public Boolean getStatus() {
        return status;
    }

    public void setStatus(Boolean status) {
        this.status = status;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String getStatusCode() {
        return statusCode;
    }

    public void setStatusCode(String statusCode) {
        this.statusCode = statusCode;
    }

    public Integer getData() {
        return data;
    }

    public void setData(Integer data) {
        this.data = data;
    }

}

File: autofin/src/main/java/v2/model/response/AddLeadResponse.kt
package v2.model.response

import com.google.gson.annotations.SerializedName


data class AddLeadResponse(
        @SerializedName("data")
        var mData: Int?,
        @SerializedName("message")
        var message: String?,
        @SerializedName("status")
        var SerializedName: Boolean?,
        @SerializedName("statusCode")
        var statusCode: String?
)